src/architecture/merkledb.md

```diff
@@ -191,8 +185,8 @@ The following additional procedures are implemented:
 
 #### ProofMapIndex
 
-[`ProofMapIndex`][proof-map-index] is a merkelized version of a map
-based on the binary Merkle Patricia tree.
+`ProofMapIndex` is a merkelized version of a map based on a binary
```

Comment:
Should we mention raw proof maps here?

Reply:
IMO, it would be going into unnecessary detail.